### PR TITLE
Home company component

### PR DESCRIPTION
**Pull request by ClaraFelix**

### CODE REVIEW

**Review by dev-luizf — COMMENTED** [review]
I noticed that the cards buttons are not making sense and there is no pattern between cards from home page and the others from hiring page and courses page. I will discuss with Eli how we can refactor them and in the meantime I will mark this card with a label indicating that it is not able to merge because of an impediment.

**Comment by ClaraFelix** [comment]
> I noticed that the cards buttons are not making sense and there is no pattern between cards from home page and the others from hiring page and courses page. I will discuss with Eli how we can refactor them and in the meantime I will mark this card with a label indicating that it is not able to merge because of an impediment.

Right, Luiz! Thank you :)

**Review by dev-luizf — COMMENTED** [review]
Eliane updated the cards:

Captura de tela de 2024-03-12 08-39-52

**Review by dev-luizf — CHANGES_REQUESTED** [review]
1. The invite button in courses cards doesn't exist anymore.
2. I'm not sure if the correct behavior of the actions menu is to open more than one or close when another one is opened. We should confirm with Eliane:

Captura de tela de 2024-03-13 14-47-45

3. This animation is a bit wierd. The actions menu should be in front of all elements:

https://github.com/Venture-Miner/wagademy/assets/83927877/91afff0b-c4c3-4aa4-8a18-4edf0bb12c2d

**Comment by dev-luizf** [comment]
> 2. I'm not sure if the correct behavior of the actions menu is to open more than one or close when another one is opened. We should confirm with Eliane:

Eliane confirmed that you must open one at a time.

**Review by Bidfca — APPROVED** [review]

File: apps/frontend/src/app/pages/company/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
```
Comment (Bidfca) [inline apps/frontend/src/app/pages/company/home/home.component.html:1]: Just noticed this problem in the responsivity when the page is 518px wide in my monitor
image

Comment (Bidfca) [reply apps/frontend/src/app/pages/company/home/home.component.html:1]: Try opening developer tools and shrink slowly the page in your device toolbar so you can notice

**Review by Bidfca — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/pages/company/home/home.component.html (hunk 1 above)
Comment (ClaraFelix) [reply apps/frontend/src/app/pages/company/home/home.component.html:1]: Right, Bia! I adjusted now :)

**Review by dev-luizf — APPROVED** [review]

**Pull request merged**